src/firefly/js/ui/FitsDownloadDialog.jsx

```diff
@@ -334,7 +350,7 @@ function resultsSuccess(request, plot) {
 
     var band = Band.NO_BAND;
     if (bandSelect) {
-        band = Band[bandSelect];
+        band = Band[bandSelect.toUpperCase()];
```

Comment:
Since Band is an Enum object, the right way to do this line is:

`band= Band.get(bandSelect)`

That converts the string (case insensitive) into the enum